core/tests/test_storage.cache.py

```
@@ -293,20 +232,21 @@ def test_get_set_bool(self):
+            orig_fields = session_1.fields
             session_1.fields = (0,) + session_1.fields[1:]
 
             # with self.assertRaises(AssertionError) as e:
             session_1.set_bool(KEY_BOOL, True)
             self.assertEqual(session_1.get_bool(KEY_BOOL), True)
 
             session_2 = cache_thp.create_or_replace_session(
-                channel.channel_cache, b"\x02"
+                channel.channel_id_bytes(), b"\x02"
             )
             session_2.fields = session_2.fields = (0,) + session_2.fields[1:]
```

Ruff (0.15.12)

[warning] Consider `(0, *session_2.fields[1:])` instead of concatenation

Replace with `(0, *session_2.fields[1:])`

(RUF005)